src/DDTrace/Integrations/Kafka/KafkaIntegration.php

```diff
@@ -0,0 +1,233 @@
+<?php
+
+namespace DDTrace\Integrations\Kafka;
+
+use DDTrace\HookData;
+use DDTrace\Integrations\Integration;
+use DDTrace\SpanLink;
+use DDTrace\Tag;
+use DDTrace\Type;
+use DDTrace\Util\ObjectKVStore;
+
+class KafkaIntegration extends Integration
+{
+    const NAME = 'kafka';
+
+    const METADATA_MAPPING = [
+        'metadata.broker.list' => Tag::KAFKA_HOST_LIST,
+        'group.id' => Tag::KAFKA_GROUP_ID,
+        'client.id' => Tag::KAFKA_CLIENT_ID
+    ];
+
+    public function init(): int
+    {
+        if (strpos(phpversion('rdkafka'), '6.') !== 0) {
```

Comment:
```suggestion
        if (strtok(phpversion('rdkafka'), '.') >= 6) {
```
Can we simply accept 6+ and adjust if needed?